### ISSUE TITLE
[JUnit StatusBar] Shows wrong progress information when the same test is executed multiple times

### ISSUE THREAD

**Issue by rherrmann** [rherrmann/eclipse-extras#49]
Even though this is a weird test suite setup, I've seen this in the wild: 

``` java
@RunWith(Suite.class)
@SuiteClasses({ SubTestSuite.class, FooTest.class })
public class MasterTestSuite {
}
```

``` java
@RunWith(Suite.class)
@SuiteClasses({ FooTest.class })
public class SubTestSuite {

}
```

``` java
public class FooTest {
    @Test
    public void testSomething() {
        fail("Not yet implemented");
    }
}
```

Running the `MasterTestSuite` will execute `testSomething` twice. The JUnit progress meter, however, will show `1 / 2`, indicating the one of two tests have been executed.

**Comment by rherrmann** [comment]
Changes reverted with commit 37cfe7b1183b9c0181ae62240d6d8d442caf8f50

**Comment by rherrmann** [comment]
Fixed by reverting commit edb9f2f81ba34c3a103477667a43ac461f723fe0 with commit 37cfe7b1183b9c0181ae62240d6d8d442caf8f50.
